===================================================================
RCS file: /cvs/gnome/banshee/src/Banshee.Base/Utilities.cs,v
retrieving revision 1.12
retrieving revision 1.13
diff -u -r1.12 -r1.13
--- src/Banshee.Base/Utilities.cs	6 Feb 2006 21:30:18 -0000	1.12
+++ src/Banshee.Base/Utilities.cs	6 Feb 2006 23:53:44 -0000	1.13
@@ -137,7 +137,7 @@
         
         public static void SetProcessName(string name)
         {
-            if(prctl(15 /* PR_SET_NAME */, Encoding.ASCII.GetBytes(name), 0, 0, 0) != 0) {
+            if(prctl(15 /* PR_SET_NAME */, Encoding.ASCII.GetBytes(name + "\0"), 0, 0, 0) != 0) {
                 throw new ApplicationException("Error setting process name: " + 
                     Mono.Unix.Native.Stdlib.GetLastError());
             }
